pkgs/top-level/aliases.nix

```diff
@@ -838,9 +838,7 @@ mapAliases {
   nixUnstable = throw "nixUnstable has been removed. For bleeding edge (Nix master, roughly weekly updated) use nixVersions.git, otherwise use nixVersions.latest."; # Converted to throw 2024-04-22
   nix_2_3 = nixVersions.nix_2_3;
   nixfmt = lib.warn "nixfmt was renamed to nixfmt-classic. The nixfmt attribute may be used for the new RFC 166-style formatter in the future, which is currently available as nixfmt-rfc-style" nixfmt-classic; # Added 2024-03-31
-
-  # When the nixops_unstable alias is removed, nixops_unstable_minimal can be renamed to nixops_unstable.
-
+  nixos-generators = throw "nixos-generators has been upstreamed. Use 'nixos-rebuild build-image' instead."; # Added 2024-11-18
```

Comment:
If I read this PR correctly, we don't provide all the functionalities of `nixos-generators` yet? For example, `proxmox-lxc`? Also keeping it around doesn't do any harm.

Reply:
True! I put that in when I had a port of the nixos-generate cli from nixos-generators added, but decided it's not really worth the effort for now.

So lets keep upstream nixos-generators for the time being :)